js/cardview.js

```diff
@@ -4,19 +4,22 @@ require("../scss/style.scss");
 
 let Card = React.createClass({
   render() {
-    var cardState = this.props.active ? 'playing-card active-card' : 'playing-card';
-    console.log(cardState)
+    
+    var cssClassForCard = this.props.hasFocus ? 'playing-card active-card' : 'playing-card';
```

Comment:
FYI in ES6 you can use "let" instead of "var" for this sort of thing, when you don't want the variable to be visible in any scopes other than the one you defined it in. That is slightly safer and therefore encouraged